src/main/java/com/homework/week15/sql/generate/SQLGenerator.java

```diff
@@ -0,0 +1,63 @@
+package com.homework.week15.sql.generate;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class SQLGenerator {
+
+    private List<Field> modelFields = new ArrayList<>();
```

Comment:
because of these global fields a user can not generate sql more than 1 time. Also it is not thread safe because of global fields again